### PR TITLE
updated metadata for current templates dev -> master

### PR DESCRIPTION
**Pull request by KiriHoy**

### CODE REVIEW

File: docs/form/index.html [hunk 1]
```diff
@@ -16,6 +16,42 @@
...
+	<meta name="twitter:title" content="Australian Government Design System" />
+	<meta name="twitter:description" content="Form pages ask users a question and allow them to answer it" />
+	<meta name="twitter:site" content="@DTA" />
+	<meta name="twitter:image" content="https://designsystem.gov.au/assets/favicons/designsystem.jpg" />
+	
+	<!-- OPENGRAPH -->
+	<meta property="og:type" content="website" />
+	<meta property="og:title" content="Form page template - Australian Government Design System" />
+	<meta property="og:site_name" content="Australian Government Design System" />
+	<meta property="og:description" content="Form pages ask users a question and allow them to answer it" />
```
Comment (sukhrajghuman) [inline docs/form/index.html]: I know this is what we have in designsystem.gov.au, but we could do something better here 😅

How about:
Forms are used to collect information from the end users with regards to a specific topic.

File: docs/index.html [hunk 2]
```diff
@@ -15,7 +15,44 @@
...
+	<meta name="robots" content="index, follow" />
+	<meta name="author" content="Digital Transformation Agency" /> 
+	<meta name="description" content="Home page template - Australian Government Design System" />
+	
+	<!-- TWITTER CARD -->
+	<meta name="twitter:card" content="summary_large_image" />
+	<meta name="twitter:title" content="Australian Government Design System" />
+	<meta name="twitter:description" content="Home page template - Australian Government Design System" />
+	<meta name="twitter:site" content="@DTA" />
+	<meta name="twitter:image" content="https://designsystem.gov.au/assets/favicons/designsystem.jpg" />
```
Comment (sukhrajghuman) [inline docs/index.html]: Should we host these assets in the starter repo? Right now if we make a change to the DS repo, it could break things in this repo

**Comment by sukhrajghuman** [comment]
nice work! @KiriHoy , just a few small things to reconsider?

File: docs/index.html (hunk 2 above)
Comment (gordongrace) [reply docs/index.html]: Good point - I'd suggest aiming to make the starter repo as self-contained as possible. Adding assets to the repo and making URL references relative could be a good way to start.

File: docs/content/index.html [hunk 3]
```diff
@@ -15,6 +15,44 @@
...
+	<meta name="robots" content="index, follow" />
+	<meta name="author" content="Digital Transformation Agency" />
+	<meta name="description" content="Content pages display information clearly" />
+	
+	<!-- TWITTER CARD -->
+	<meta name="twitter:card" content="summary_large_image" />
+	<meta name="twitter:title" content="Australian Government Design System" />
+	<meta name="twitter:description" content="Content pages display information clearly" /> 
+	<meta name="twitter:site" content="@DTA" />	
+	<meta name="twitter:image" content="../assets/favicons/designsystem.jpg" />
```
Comment (gordongrace) [inline docs/content/index.html]: Bit of a tough one - OG and Twitter image metadata expect content values to be full URLs, complete with protocols. Relative paths aren't officially supported.

https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary-card-with-large-image

File: docs/index.html (hunk 2 above)
Comment (sukhrajghuman) [reply docs/index.html]: Yep good point

We should do 
https://govau.github.io/design-system-starter/assets/favicons/designsystem.jpg

**Pull request merged**